src/endpoint.ts

```diff
@@ -287,30 +273,16 @@ export class Endpoint<
     response,
     logger,
     config,
-    siblingMethods = [],
   }: {
     request: Request;
     response: Response;
     logger: ActualLogger;
     config: CommonConfig;
-    siblingMethods?: Method[];
   }) {
     const method = getActualMethod(request);
     const options: Partial<OPT> = {};
     let output: FlatObject | null = null;
     let error: Error | null = null;
-    if (config.cors) {
-      let headers = this.#getDefaultCorsHeaders(siblingMethods);
-      if (typeof config.cors === "function") {
-        headers = await config.cors({
-          request,
-          logger,
-          endpoint: this,
-          defaultHeaders: headers,
-        });
-      }
-      for (const key in headers) response.set(key, headers[key]);
```

Comment:
this might be breaking for `testEndpoint({ method: "OPTIONS" })`.
should address to v21

Reply:
rebased